docs/en/integrations/language-clients/js.md

```diff
@@ -248,14 +268,56 @@ for await (const rows of resultSet.stream()) {
 }
 ```
 
+**Example:** (Web only) Iteration over the `ReadableStream` of objects
+
+
 ### Insert method
 
-The primary method for data insertion. It can work with both `Stream.Readable` (all formats except `JSON`) and
+The primary method for data insertion.
+Does not return anything aside from `query_id` - the response stream is immediately destroyed.
+
+```ts
+export interface InsertResult {
+  query_id: string
+}
+
+interface ClickHouseClient {
+  insert(params: InsertParams): Promise<InsertResult>
+}
+```
+
+#### Web version limitations
+
+Currently, `@clickhouse/client-web` does not support streaming for `insert` operation 
```

Comment:
Let's change the order: first - what's supported, second - what's not supported.